docs/actors/aws.iam.UploadCert.md

```diff
@@ -0,0 +1,32 @@
+##### kingpin.actors.aws.iam.UploadCert
+
+Uploads a new SSL Cert to AWS IAM.
+
+    http://boto.readthedocs.org/en/latest/ref/iam.html
+    #boto.iam.connection.IAMConnection.upload_server_cert
+    
+**Options**
+
+* `private_key_path` - str: Path to the private key.
+* `path` - str: The path for the server certificate. Optional.
```

Comment:
From the documentation, I'm confused about these options. What is the `path`? How is it different than the `public_key_path`?